CONTRIBUTING.md

```diff
@@ -1,203 +1,332 @@
+
+If you wish to work on an open issue, please comment with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. So, assign yourself to an issue before beginning work to avoid conflicts.
+
+When you contribute to the project for the first time, please consider checking the [bug](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22%F0%9F%90%9B+bug%22) or [good first issue](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) labels.
 
-If you are contributing to the project for the first time, please consider checking the [bug](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22%F0%9F%90%9B+bug%22) or [good first issue](https://github.com/open-sauced/intro/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) labels.
+If you get stuck, you can always ask for help in the [Discord](https://discord.com/invite/U2peSNf23P) or GitHub Discussions.
 
-In case you get stuck, please feel free to ask for help in the [Discord](https://discord.com/invite/U2peSNf23P) server or GitHub Discussions.
+## Getting Started
 
-## Setup the Project Locally
-1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [intro repository](github.com/open-sauced/intro) you are working on  to your own GitHub account.
+### Setup the Project Locally
+
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [intro repository](github.com/open-sauced/intro) to your own GitHub account.
 2. [Clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) the forked repository to your local machine.
-```bash
-git clone https://github.com/your-username/intro.git
-```
-3. cd into the repository and open the project in your code editor.
-```bash
-cd intro
-```
-4. Run the course locally. See [Running the Project Locally](#running-the-project-locally) for more information. 
+
+   ```bash
+   git clone https://github.com/<your-username>/intro.git
+   ```
+
+3. Navigate to and open the project in your code editor.
+
+   ```bash
+   cd intro
+   ```
+
+4. Run the project.
 
 ### Running the Project Locally
-To contribute effectively to the OpenSauced project, it's essential to run the project locally to see your changes in action and test them thoroughly. We recommend two methods for running the project on your local machine: using the Docsify CLI or leveraging Visual Studio Code's Go Live feature.
+
+It's essential to run the project locally to see your changes in real-time and test them thoroughly when you contribute to this project.
+
+We recommend two methods for running the project on your local machine:
```

Comment:
[suggestion] It'll be best to clarify to contributors to use only one of the methods so they won't end up using both of them. 
```suggestion
We recommend using one of the two methods for running the project on your local machine:
```